### PR TITLE
Version mention updates v1.1

### LINKED ISSUE #1341: Update version mentions in branch v1.1
**What content needs to be created or modified?**

Need to update "current" version in v1.1 branch to v1.1

**Describe the solution you'd like**

N/A

**Where should the new material be placed?**

- In getting started docs
-  Upgrade guidance
- main README for branch guidance
- Support section

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by orizohar**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

- Updating references to current version to be v1.1
- Updating support tables
- Updating branch guidance

## Issue reference

Closes #1341

### CODE REVIEW

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status |
 |--------------------|:--------:|:--------|---------|---------|---------|
-| Feb 17th 2021 | 1.0.1</br>(Mar 4th 2021) | 1.0.0 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported (current) |
+| Feb 17th 2021 | 1.0.0</br>| 1.0.0 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported |
+| Mar 4th 2021  | 1.0.1</br>| 1.0.1 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported |
```
Comment (orizohar) [inline daprdocs/content/en/operations/support/support-release-policy.md]: I split the v1.0.0 and v1.0.1 to two separate rows

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
...
+| Mar 30th 2021 | 1.1.0</br> | 1.1.0 | Java 1.1.0 </br>Go 1.1.0 </br>PHP 1.1.0 </br>Python 1.1.0 </br>.NET 1.1.0 | 0.6.0 | Supported (current) |
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/support/support-release-policy.md]: Java it is still 1.0.2 ... not 1.1.0 ...

**Review by mukundansundar — COMMENTED** [review]
@artursouza the java sdk version should be 1.0.2 right?

File: daprdocs/content/en/operations/support/support-release-policy.md (hunk 2 above)
Comment (orizohar) [reply daprdocs/content/en/operations/support/support-release-policy.md]: Thanks, I wasn't sure about the SDK version. 

@wcs1only @rynowak @skyao @withinboredom can you provide the correct versions for the Python, .NET, Go and PHP SDKs that work with v1.1 respectively?

Comment (rynowak) [reply daprdocs/content/en/operations/support/support-release-policy.md]: new .NET SDK release will be 1.1.0, however 1.0.0 should still be working

**Review by artursouza — APPROVED** [review]

**Pull request merged**